lib/content-services/content-node-selector/content-node-dialog.service.spec.ts

```diff
@@ -103,28 +104,100 @@ describe('ContentNodeDialogService', () => {
         spyOn(documentListService, 'getFolderNode').and.returnValue(Promise.resolve(fakeNode));
         service.openFileBrowseDialogBySite().subscribe();
         tick();
-        expect(materialDialog.open).toHaveBeenCalled();
+        expect(spyOnDialogOpen).toHaveBeenCalled();
     }));
 
     it('should be able to open the dialog for folder using the first user site', fakeAsync(() => {
         spyOn(sitesService, 'getSites').and.returnValue(Observable.of(fakeSiteList));
         spyOn(documentListService, 'getFolderNode').and.returnValue(Promise.resolve(fakeNode));
         service.openFolderBrowseDialogBySite().subscribe();
         tick();
-        expect(materialDialog.open).toHaveBeenCalled();
+        expect(spyOnDialogOpen).toHaveBeenCalled();
     }));
 
     it('should be able to close the material dialog', () => {
         service.close();
         expect(materialDialog.closeAll).toHaveBeenCalled();
     });
 
+    describe('for the copy/move dialog', () => {
+        const siteNode: MinimalNodeEntryEntity = <MinimalNodeEntryEntity> {
+            id: 'site',
+            name: 'siteNode',
+            nodeType: 'st:site'
+        };
+        const sites: MinimalNodeEntryEntity = <MinimalNodeEntryEntity> {
+            id: 'sites',
+            name: 'sites',
+            nodeType: 'st:sites'
+        };
+        const site: MinimalNodeEntryEntity = <MinimalNodeEntryEntity> {
+            id: 'site-id',
+            name: 'site',
+            guid: 'any-guid'
+        };
+        const nodeEntryWithRightPermissions: MinimalNodeEntryEntity = <MinimalNodeEntryEntity> {
+            id: 'node-id',
+            name: 'nodeWithRightPermissions',
+            allowableOperations: ['create']
+        };
+        const nodeEntryNoPermissions: MinimalNodeEntryEntity = <MinimalNodeEntryEntity> {
+            id: 'node-id',
+            name: 'nodeWithNoPermissions',
+            allowableOperations: []
+        };
+
+        const fixture = [
+            {
+                node: siteNode,
+                infoKey: 'nodeType',
+                expected: false
+            },
+            {
+                node: sites,
+                infoKey: 'nodeType',
+                expected: false
+            },
+            {
+                node: site,
+                infoKey: 'guid',
+                expected: false
+            },
+            {
+                node: nodeEntryWithRightPermissions,
+                infoKey: 'allowableOperations',
+                expected: true
+            },
+            {
+                node: nodeEntryNoPermissions,
+                infoKey: 'allowableOperations',
+                expected: false
+            }
+        ];
+
+        fixture.forEach((testData) => {
+            it(`should ${testData.expected ? '' : 'NOT '}allow selection for ${testData.node.name} (${testData.infoKey} = \'${testData.node[testData.infoKey]}\')`, () => {
```

Comment:
is not clear to me the behaviour this seems to me more descibing how the code woeks